mne/viz/evoked.py

```diff
@@ -126,11 +128,33 @@ def _rgb(x, y, z):
     return np.asarray([x, y, z]).T
 
 
+def _plot_legend(pos, colors, axis, outlines='skirt'):
```

Comment:
Just an idea: 'head' might also in some cases be preferable over 'skirt'. `outlines` could also be set via the argument of `legend`. If True, set to 'skirt', if 'head', use 'head'. I think 'skirt' is not the default for plot_topomap.

So like,
```
outlines = outlines if isinstance(outlines, str) else 'skirt'
if outlines not in ["head", "skirt"]:
    raise ValueError: "`outlines` should be bool, 'skirt' or 'head'"
```